content/source/docs/enterprise/saml/identity-provider-configuration-forgerock.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: "enterprise"
+page_title: "SAML ForgeRock Identity Provider Configuration - Terraform Enterprise"
+---
+
+# ForgeRock Configuration
+
+Follow these steps to configure ForgeRock as the identity provider (IdP) for Terraform Enterprise.
+
+## Configure a new Remote Service Provider in ForgeRock
+
+On the TFE console, paste the ForgeRock Access Manager hosted IDP metadata certificate in the proper pem encoded format.  Enter the IDP SSO and SLO URLs as well.
+
+
+In the hosted IDP assertion content tab the highlighted entry must be on the top to be first in the order of precedence.
```

Comment:
Need to name the desired entry in the text of the instructions. Can't just screenshot it, that's not accessible.